snippets/matter-debug/snippet.yml

```diff
@@ -0,0 +1,23 @@
+name: matter-debug
+append:
+  EXTRA_CONF_FILE: matter_debug.conf
+
+boards:
+  nrf52840dk/nrf52840:
```

Comment:
could use wildcards to also support customer boards with these socs

Reply:
Sure thing, good point.

Reply:
Oh, I've noticed now that it cannot be done by using wildcards. We want to change the UART speed in those .dts overlays, so it is applicable only for DKs. Customers must add their own dts overlay for UART if they want to increase the speed because they can use a different UART than is used on the DK. I will open a follow-up and describe it in our docs as well.